### PR TITLE
PlayerHighlightFactor and PlayerHighlightComponent Code Coverage

### PR DESCRIPTION
**Pull request by 86LAK**
Wrote more junit testing for PlayerHighlightFactory and the PLayerHighlightCOmponent to boost code coverage.

### CODE REVIEW

File: source/core/src/test/com/csse3200/game/entities/factories/PlayerHighlightFactoryTest.java [hunk 1]
```diff
@@ -32,4 +32,13 @@ public void testCreatePlayerHighlight() {
         DynamicTextureRenderComponent renderComponent = playerHighlight.getComponent(DynamicTextureRenderComponent.class);
         Assertions.assertNotNull(renderComponent);
     }
+
+    /**
+     * Tests that the PlayerHighlightFactory class can be created
+     */
```
Comment (JTrenerry) [inline source/core/src/test/com/csse3200/game/entities/factories/PlayerHighlightFactoryTest.java:38]: ???

**Review by JTrenerry — APPROVED** [review]
LGTM

File: source/core/src/test/com/csse3200/game/entities/factories/PlayerHighlightFactoryTest.java (hunk 1 above)
Comment (86LAK) [reply source/core/src/test/com/csse3200/game/entities/factories/PlayerHighlightFactoryTest.java:38]: What

Comment (JTrenerry) [reply source/core/src/test/com/csse3200/game/entities/factories/PlayerHighlightFactoryTest.java:38]: What does that do?

Comment (86LAK) [reply source/core/src/test/com/csse3200/game/entities/factories/PlayerHighlightFactoryTest.java:38]: Checks it Can be instantiated

**Pull request merged**